locale/ar/get-involved/node-events.md

```diff
@@ -0,0 +1,20 @@
+﻿---
+title: مناسبات الـ Node.js
+layout: contribute.hbs
+---
+
+# مناسبات الـ Node.js
+
+هذه قائمة بالمناسبات و المؤتمرات الخاصة بالـ Node.js. قم بإيداع طلب تعديل إذا أردت إضافة مناسبة جديدة لم تتم إضافتها بعد إلى القائمة أسفله.
+
+## ملاحظات بخصوص إضافة المناسبات
+
+الشكل
+- [إسم الحدث](https://) - المكان، التاريخ
+
+## المناسبات
+
+### 2019
+- [NodeConf Colombia](https://colombia.nodeconf.com) - مدلين، كولومبيا 21 و 22 جوان 
```

Comment:
جوان  > يونيو

Reply:
> 
> 
> Hi @celyes：
> Like 'node-speakers.md', you DON'T need to translate 'node-events.md' because this file MUST BE submitted in English, on the other hand, this file will be frequently changed.
> 
> For related things, please refer to this: #2022
Thank you so much @Maledong , but since it's OK for now as @fhemberger said in #2022, we will keep it updated in Arabic every time the English version gets an update.